alacritty/src/display.rs

```diff
@@ -4,6 +4,7 @@
 use std::cmp::min;
 use std::f64;
 use std::fmt::{self, Formatter};
+use std::mem;
```

Comment:
Since the function you're using from it is used like once, maybe we can save us one line and use `std::mem::swap`? Though, I don't really mind the `use`.

Reply:
Generally we always import things even if it's just used once, which I think is a good idea because that means you don't have to go hunting for it the next time you add something from `mem` (you'll see immediately that there's an import already.

So I really wouldn't want to remove the import here unless there's a clear advantage to using the full path (like saving yourself a #[cfg()] for example if it's in a method that only exists on one platform).